dash.js

```diff
@@ -69,19 +68,20 @@ const MyDashActor = new Lang.Class({
             orientation: this._isHorizontal ? Clutter.Orientation.HORIZONTAL : Clutter.Orientation.VERTICAL
         });
 
-        this.parent({
+        super._init({
             name: 'dash',
             layout_manager: layout,
             clip_to_allocation: true
         });
+        this.name = this.constructor.name;
```

Comment:
@3v1n0 This seems to create some styling problems, as the widget loses thr `#Dash` id/name and things like the dash background are not set. Any reason why this might be necessary?

Reply:
From my tests and understanding it is not necessary (a4cb6420bf07e5639).

Reply:
Thanks for catching it!